src/main/kotlin/no/nav/familie/klage/henlegg/HenleggBehandlingController.kt

```diff
@@ -0,0 +1,81 @@
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.util.UUID
+
+@RestController
+@RequestMapping(path = ["/api/behandling"])
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class HenleggBehandlingController(
+    private val tilgangService: TilgangService,
+    private val henleggBehandlingService: HenleggBehandlingService,
+    private val brevService: BrevService,
+    private val personopplysningerService: PersonopplysningerService,
+) {
+
+    @PostMapping("{behandlingId}/henlegg")
+    fun henleggBehandling(@PathVariable behandlingId: UUID, @RequestBody henlegg: HenlagtDto): Ressurs<Unit> {
+        tilgangService.validerTilgangTilPersonMedRelasjonerForBehandling(behandlingId, AuditLoggerEvent.UPDATE)
+        tilgangService.validerHarSaksbehandlerrolleTilStønadForBehandling(behandlingId)
+        if (henlegg.skalSendeHenleggelsesbrev) {
+            validerIkkeSendBrevPåFeilType(henlegg)
+            validerIkkeSendBrevHvisVergemålEllerFullmakt(behandlingId)
+            brevService.opprettJournalførHenleggelsesbrevTask(behandlingId)
+        }
+        return Ressurs.success(henleggBehandlingService.henleggBehandling(behandlingId, henlegg))
+    }
+
+    @GetMapping("/{behandlingId}/henlegg/brev/forhandsvisning")
+    fun genererHenleggBrev(
+        @PathVariable behandlingId: UUID,
+    ): Ressurs<ByteArray> {
+        tilgangService.validerTilgangTilPersonMedRelasjonerForBehandling(behandlingId, AuditLoggerEvent.UPDATE)
+        tilgangService.validerHarSaksbehandlerrolleTilStønadForBehandling(behandlingId)
+        return henleggBrevRessurs(behandlingId)
+    }
+
+    private fun henleggBrevRessurs(
+        behandlingId: UUID,
+    ) = Ressurs.success(brevService.genererHenleggelsesbrev(behandlingId))
+
+    private fun validerIkkeSendBrevPåFeilType(henlagt: HenlagtDto) {
+        feilHvis(henlagt.skalSendeHenleggelsesbrev && henlagt.årsak == HenlagtÅrsak.FEILREGISTRERT) { "Skal ikke sende brev hvis type er ulik trukket tilbake" }
+    }
+
+    private fun validerIkkeSendBrevHvisVergemålEllerFullmakt(
+        ident: UUID,
+    ) {
+        val personopplysninger = personopplysningerService.hentPersonopplysninger(ident)
+        val harVerge = personopplysninger.vergemål.isNotEmpty()
+        val harFullmakt: Boolean =
+            personopplysninger.fullmakt
+                .filter {
+                    it.gyldigTilOgMed == null ||
+                        (
+                            it.gyldigTilOgMed.isEqualOrAfter(
+                                LocalDate.now(),
+                            )
+                            )
+                }.isNotEmpty()
+        feilHvis(harVerge || harFullmakt) {
+            "Skal ikke sende brev hvis person er tilknyttet vergemål eller fullmakt"
+        }
```

Comment:
Nå blir det error-meldinger i loggene/på slack dersom dette oppstår - er det ønskelig? 
Er det tenkt å støtte manuelle brevmottakere for henleggelsesbrev, eller løser de det "manuelt" ? 

Tror feilmeldingen bør sees på - usikker på om saksbehandlere skjønner hva de skal gjøre dersom de får denne meldingen?